src/components/App.js

```diff
@@ -1,16 +1,161 @@
 import React from 'react';
+import Menu from './Menu.js'
+import Header from './Header.js'
+import SeeOrder from "./SeeOrder.js"
+import OrderReview from "./OrderReview.js"
+import OrderConfirmation from "./OrderConfirmation.js"
 
 import '../styles/App.scss';
 
 class App extends React.Component {
   constructor(){
     super();
+
+    this.getMenu = this.getMenu.bind(this)
+    this.addToOrder = this.addToOrder.bind(this)
+    this.removeFromOrder = this.removeFromOrder.bind(this)
+    this.amendQuantity = this.amendQuantity.bind(this)
+    this.changeDisplay = this.changeDisplay.bind(this)
+    this.placeOrder = this.placeOrder.bind(this)
+
+    this.state={
+      menu: {},
+      currentOrder: {
+        total: 0,
+        items: {}
+                    },
+      recentOrder: {},
+      display: 'menu'  //'menu' or 'order' or 'confirmation'
+
+    }
+  }
+
+  componentDidMount(){
+    this.getMenu()
+  }
+
+
+  getMenu(){
+    fetch('/api/menu')
+      .then(response => response.json())
+      .then(body => {
+        this.setState({
+          menu: body
+        })
+      })
+  }
+
+  placeOrder(){
+    const order = this.state.currentOrder
+    fetch('/api/order', {
+      method: 'post',
+      body: JSON.stringify(order),
+      headers: {
+        'Content-Type': 'application/json'
+      }
+    })
+    .then(response => response.json())
+    .then(body => {
+      console.log(body)
+      this.changeDisplay('confirmation')
+      this.setState({
+        recentOrder: body,
+        currentOrder: {
+        total: 0,
+        items: {},
+      }
+      })
+    })
+  }
+
+  addToOrder(menuItem, quantity){
+    if (!this.state.currentOrder.items[menuItem.id]){ //if item not already in order, add to currentOrder
+    const foodItem = {
+      [menuItem.id]: {
+        quantity,
+        totalPrice: quantity * menuItem.price,
+        menuItem
+      }
+    }
+
+    const currentOrder = {
+      total: this.state.currentOrder.total + foodItem[menuItem.id].totalPrice,
+      items: Object.assign(this.state.currentOrder.items, foodItem)
```

Comment:
IT would be better to us `Object.assign({}, this.state.currentOrder.items, foodItem)` as the current implementation mutates state